project-icp/bee-pollinator-app
# Pin to pip 18.1

## Overview

Between Jenkins builds [#120](
http://civicci01.internal.azavea.com/job/bee-pollinator-app-and-worker/120/) and [#121](
http://civicci01.internal.azavea.com/job/bee-pollinator-app-and-worker/121/), `pip 19.0` was released (https://pypi.org/project/pip/#history).

We weren't pinning to a specific version of `pip` in the Packer shell provisioner, so builds started using `pip 19.0`, and builds started failing.

Fixes #441 

## Testing Instructions

See successful CI build: http://civicci01.internal.azavea.com/job/bee-pollinator-app-and-worker/129/
